torchsde/_brownian/brownian_interval.py

```diff
@@ -477,23 +477,18 @@ def __call__(self, ta, tb=None, return_U=False, return_A=False):
             if len(intervals) > 1:
                 # If we have multiple intervals then add up their increments and Levy areas.
 
-                # Clone to avoid modifying the W, H, A that may exist in the cache
-                W = W.clone()
-                if self.have_H:
-                    H = H.clone()
-                if self.have_A:
-                    A = A.clone()
-
                 for interval in intervals[1:]:
                     Wi, Hi, Ai = interval.increment_and_levy_area()
                     if self.have_H:
-                        H += Hi + (interval.end - interval.start) * W
+                        term1 = (interval.end - interval.start) * (Hi + 0.5 * W)
```

Comment:
I double checked the revised formula and they look correct. 

But just to record things, could we type out the formula for aggregation in comments? Something like H_{s,t} = ... H_{s,u} ... H_{u, t} ... W_{s, u} ... 

Might be worthwhile to do the same for A.

Reply:
Done.